### PR TITLE
Working example for the most common case: Flask

### PR DESCRIPTION
**Pull request by mdamien**
Maybe the wording isn't there but I modified the `python.json` so it a least looks like something that's gonna work :smile: .

### CODE REVIEW

**Comment by judu** [comment]
Can I react on the wording? I'll let you fix and update the PR :)

File: runtimes/python/python_apps.md [hunk 1]
```diff
@@ -57,12 +57,14 @@ To select which module you want to start, use the key `module` in `deploy` like
...
    {
       "deploy": {
-         "module": "myapplication"
+         "module": "mymodule:app"
       }
    }
 ```
 
-The module (without .py) must be importable, i.e. be in `PYTHONPATH`. For example with Django: "module":"cc_django_wsgi".
+The module (without .py) must be importable, i.e. be in `PYTHONPATH`. For example with Flask, it's gonna be the name of your main server file followed by your Flask object: `server:app` for exemple if you have a `server.py` file at the root of your project with a Flask `app` object inside.
```
Comment (judu) [inline runtimes/python/python_apps.md]: exemple -> example.

File: runtimes/python/python_apps.md [hunk 2]
```diff
@@ -57,12 +57,14 @@ To select which module you want to start, use the key `module` in `deploy` like
...
+
+As a generality, you should just point to a WSGI capable object.
```
Comment (judu) [inline runtimes/python/python_apps.md]: "As a generality" -> "Basically"

**Comment by mdamien** [comment]
Updated it (maybe you didn't get the notif)

**Pull request merged**